src/lib/config/mainnet/pools.ts

```diff
@@ -104,6 +104,8 @@ const pools: Pools = {
       '0x0bbc7b78ff8453c40718e290b33f1d00ee67274e000000000000000000000563', // baoeth/beth
       '0x911fc393355fe415a667893d55fff6703c57c72e000200000000000000000574', // 50ETHx-50WETH
       '0xd8721e92ba0f8235b375e9ec9a7b697ec4e2d6c6000200000000000000000575', // 80BIDS-20WETH
+      '0xbf2ef8bdc2fc0f3203b3a01778e3ec5009aeef3300000000000000000000058d', // R/bbsDAI
+      '0x4cbde5c4b4b53ebe4af4adb85404725985406163000000000000000000000595', // ETHx/WETH
```

Comment:
This pool is using a strange underlying token for `bb-a-weth` It is [stataEthWETH](https://etherscan.io/address/0x03928473f25bb2da6bc880b07ecbadc636822264) which is different from the other `bb-a-weth` pools. 

This is what http://localhost:8080/#/ethereum/pool/0x02d928e68d8f10c0358566152677db51e1e2dc8c00000000000000000000051e is using:

![normal-bb-a-weth pool](https://github.com/balancer/frontend-v2/assets/525534/01f26fd5-b68c-40c2-9d92-bf967ca78dd1)

This is what this pool is using:
![pool-composition](https://github.com/balancer/frontend-v2/assets/525534/cce3d73a-1859-4dc7-90a6-18818af28870)

Do you know what is going on here? Why is this different from other pools?